org.adempiere.base/src/org/idempiere/process/VerifyMigration.java

```diff
@@ -0,0 +1,246 @@
+ * as published by the Free Software Foundation; either version 2      *
+ * of the License, or (at your option) any later version.              *
+ *                                                                     *
+ * This program is distributed in the hope that it will be useful,     *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of      *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the        *
+ * GNU General Public License for more details.                        *
+ *                                                                     *
+ * You should have received a copy of the GNU General Public License   *
+ * along with this program; if not, write to the Free Software         *
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,          *
+ * MA 02110-1301, USA.                                                 *
+ *                                                                     *
+ * Contributors:                                                       *
+ * - Carlos Ruiz - globalqss - bxservice                               *
+ * Sponsored by FH                                                     *
+ **********************************************************************/
+
+package org.idempiere.process;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.logging.Level;
+
+import org.adempiere.exceptions.DBException;
+import org.compiere.model.MChangeLog;
+import org.compiere.model.MColumn;
+import org.compiere.model.MPInstance;
+import org.compiere.model.MTable;
+import org.compiere.model.MUser;
+import org.compiere.model.MVerifyMigration;
+import org.compiere.model.MViewColumn;
+import org.compiere.model.MViewComponent;
+import org.compiere.model.PO;
+import org.compiere.model.Query;
+import org.compiere.process.ProcessInfoParameter;
+import org.compiere.process.SvrProcess;
+import org.compiere.util.DB;
+import org.compiere.util.Msg;
+
+@org.adempiere.base.annotation.Process
+public class VerifyMigration extends SvrProcess {
+
+	// Process to help verifying after a migration, check if customizations were
+	// overwritten
+
+	/* Date To */
+	private Timestamp p_DateTo = null;
+	/* SeqNo */
+	private int m_SeqNo = 0;
+
+	@Override
+	protected void prepare() {
+		for (ProcessInfoParameter para : getParameter()) {
+			String name = para.getParameterName();
+			switch (name) {
+			case "DateTo":
+				p_DateTo = para.getParameterAsTimestamp();
+				break;
+			default:
+				if (log.isLoggable(Level.INFO))
+					log.log(Level.INFO, "Custom Parameter: " + name + "=" + para.getInfo());
+				break;
+			}
+		}
+	}
+
+	@Override
+	protected String doIt() throws Exception {
+		if (log.isLoggable(Level.INFO))
+			log.info("DateTo" + p_DateTo);
+
+		int cnt = 0;
```

Comment:
couldn't this just be one line ->

int cnt = verifyCustomizationsInChangeLog()  + verifyCustomViewColumns();

Reply:
my plan is to add later more cases - and I think it gives more readability to have the calls in separate lines

Reply:
I think it is less readable and it makes the method longer than it needs to be but as it is a minor thing, so it is not a blocker